duplicity-backup.sh

```diff
@@ -361,7 +361,7 @@ mailcmd_nail() {
   ${MAILCMD} -s "${EMAIL_SUBJECT}" -r "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
 }
 mailcmd_else() {
-  ${MAILCMD} "${EMAIL_SUBJECT}" "${EMAIL_FROM}" "${EMAIL_TO}" < "${LOGFILE}"
+  mailcmd_sendmail
```

Comment:
Thanks for the PR, but please merge or rebase your commit on top of the current `dev` so that you don't undo my latest commit 6dc6d19 (2 hours ago).